### ISSUE TITLE
Inconsistent spelling of licence

### ISSUE THREAD

**Issue by jeffcullis** [cioos-siooc/metadata-xml#92]
A data provider in the Atlantic region noted a small inconsistency in the spelling of licence in the YAML:
![image](https://user-images.githubusercontent.com/3650576/118538590-c82c8a80-b724-11eb-9c68-d1045bf2f694.png)

ISO requires 'licence' be used in its option list.

A fairly minor issue but could be worth trying to standardize on 'licence' throughout.

**Comment by n-a-t-e** [comment]
Thanks, just to clarify, the issue is with `code: government-open-license-nova-scotia`, right? I don't know how this code was made up but I think it may not be the proper code anyways. It looks like the proper code might be `OGL-C`

**Comment by n-a-t-e** [comment]
I haven't found a lot online about these provincial codes, but looks like it might be `OGL-NS`, `OGL-BC`, etc

https://developer.gov.bc.ca/Licensing-Your-Project

**Comment by jeffcullis** [comment]
Yeah that's right, just the code part. I like the idea of going with the standard OGL-C federally and OGL-NS, OGL-BC etc. for  each province.
